src/ucan/ucan-gateway.ts

```diff
@@ -287,28 +286,61 @@ export class UCANGateway implements bs.Gateway {
     return Result.Ok(undefined);
   }
 
+  private readonly subscriberCallbacks = new Set<(data: Uint8Array) => void>();
+
+  private notifySubscribers(data: Uint8Array): void {
+    for (const callback of this.subscriberCallbacks) {
+      try {
+        callback(data);
+      } catch (error) {
+        console.error(error);
+        this.logger.Error().Err(error).Msg("Error in subscriber callback execution");
+      }
+    }
+  }
+
+  async subscribe(url: URI, callback: (msg: Uint8Array) => void): Promise<bs.UnsubscribeResult> {
+    // eslint-disable-next-line
+    if (!this.inst?.polling) return Result.Ok(() => {});
```

Comment:
Note to self, subscribe function is required, even though it just returns an `Ok` with a callback that does nothing; otherwise it doesn't merge in the data from other database instances.